dashboards/pods.libsonnet

```diff
@@ -17,22 +17,23 @@ local numbersinglestat = promgrafonnet.numbersinglestat;
           'Memory Usage',
           datasource='$datasource',
           min=0,
+          span=12,
```

Comment:
Why the new specific span?

Reply:
without it, the first graph appears half-width. Span=12 also appears in persistentvolumesusage.libsonnet

Figure (before)
![image](https://user-images.githubusercontent.com/5599664/51707433-781edc80-1fde-11e9-9cac-8c2db817f3ea.png)